ide/coqide/idetop.ml

```diff
@@ -397,8 +397,8 @@ let set_options options =
 let about () = {
   Interface.coqtop_version = Coq_config.version;
   Interface.protocol_version = Xmlprotocol.protocol_version;
-  Interface.release_date = Coq_config.date;
-  Interface.compile_date = Coq_config.compile_date;
+  Interface.release_date = "n/a";
+  Interface.compile_date = "n/a";
```

Comment:
Shouldn't we remove those fields from the ocaml record type and put n/a here https://github.com/coq/coq/blob/c6bc1cea49cb5a18142437325ecb6875514c22bb/ide/coqide/protocol/xmlprotocol.ml#L237-L249 instead?

Reply:
I do not know who the users are. If people are directly using the XML protocol, then yes, we can remove these fields from the record. But if some people are using `xmlprotocol.ml` (and thus `interface.ml`) to encode/decode the protocol, then we are breaking compatibility for these users. I am completely clueless with respect to the current status of user interfaces for Coq.

Reply:
I believe protocol version is what matters, and this PR is not fiddling with it.

In general it makes no sense to write any code which tests the compilation *date* of a piece of software. The version "number" has a meaning, but day it was compiled, frankly no. Eg a distribution may decide to recompile today an old version of any package (e.g. if there was a security bug in `ocamlc` then Debian would recompile all OCaml package almost automatically, and Coq-same-version would get a new date)

Reply:
So do we change this or merge as-is?

Reply:
If we remove them we change the protocol and hence we should update the protocol version number.

I don't have a strong opinion, I'm ok with this PR as is.